pricegraph/src/orderbook.rs

```diff
@@ -110,6 +103,59 @@ impl Orderbook {
         let fee_token = node_index(0);
         bellman_ford::search(&self.projection, fee_token).is_err()
     }
+
+    /// Updates the projection graph for every token pair.
+    pub fn update_projection_graph(&mut self) {
```

Comment:
This will be called incrementally for each found path/cycle?

Reply:
No, this is called once when removing negative cycles, then we do it per-edge per path.

Reply:
It would be really bad if it had to be called at each iteration!